app/ui/client/views/app/room.html

```diff
@@ -11,8 +11,8 @@
 				{{/headerRoom}}
```

Comment:
You need to add this logic to the thread view as well https://github.com/RocketChat/Rocket.Chat/blob/fix-drag-drop/app/threads/client/flextab/thread.html#L14

Reply:
Done